This branch is to add "The Bahamas" queryPreset in presets.go 🇧🇸

steveWDamesJr opened the pull request:
## Add The Bahamas 🇧🇸 

- Please find the queryPreset "**The Bahamas**" within presets.go

Thanks for your efforts 🙏

steveWDamesJr committed:
Update presets.go with queryPreset The Bahamas

Add list of principal islands in The Bahamas

ashkulz reviewed (changes requested):
Please use `+` instead of space between words.

steveWDamesJr committed:
Update presets.go countries

Add '+' rather than space between words
-Update country names from lowercase first letters to capital letters

ashkulz commented on presets.go:
```diff
@@ -383,6 +383,9 @@ var PRESETS = map[string]QueryPreset{
 	"united states": QueryPreset{
 		include: []string{",+US", "USA", "United+States", "Alabama", ",+AL", "Alaska", ",+AK", "Arizona", ",+AZ", "Arkansas", ",+AR", "California", ",+CA", "Colorado", ",+CO", "Connecticut", ",+CT", "Delaware", ",+DE", "Florida", ",+FL", "Georgia", ",+GA", "Hawaii", ",+HI", "Idaho", ",+ID", "Illinois", ",+IL", "Indiana", ",+IN", "Iowa", ",+IA", "Kansas", ",+KS", "Kentucky", ",+KY", "Louisiana", ",+LA", ...
 	},
+	"the bahamas": QueryPreset{
+		include: []string{"The+Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
```
```suggestion
	"bahamas": QueryPreset{
		include: []string{"Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
```
Without this change, someone who just puts in "Bahamas" won't be picked up.

steveWDamesJr committed:
Add 'Bahamas' to QueryPreset

steveWDamesJr commented on presets.go:
```diff
@@ -384,7 +384,7 @@ var PRESETS = map[string]QueryPreset{
 	},
 	"the bahamas": QueryPreset{
-		include: []string{"The+Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
+		include: []string{"The+Bahamas", "Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
```
Hello @ashkulz 👋 
I hope this recent commit represents the inclusive change required to now correctly add 'The Bahamas' to the Most Active Github Users List. Thank you!

ashkulz replied:
Having `The Bahamas` and `Bahamas` is redundant, so better to just have the latter.

Also, I'm not the maintainer (@lauripiispanen is) and I can just merge this in my fork https://committers.top :man_shrugging:

steveWDamesJr committed:
Update QueryPreset in presets.go

steveWDamesJr commented on presets.go:
```diff
@@ -383,6 +383,9 @@ var PRESETS = map[string]QueryPreset{
 		include: []string{",+US", "USA", "United+States", "Alabama", ",+AL", "Alaska", ",+AK", "Arizona", ",+AZ", "Arkansas", ",+AR", "California", ",+CA", "Colorado", ",+CO", "Connecticut", ",+CT", "Delaware", ",+DE", "Florida", ",+FL", "Georgia", ",+GA", "Hawaii", ",+HI", "Idaho", ",+ID", "Illinois", ",+IL", "Indiana", ",+IN", "Iowa", ",+IA", "Kansas", ",+KS", "Kentucky", ",+KY", "Louisiana", ",+LA", ...
 	},
+	"the bahamas": QueryPreset{
+		include: []string{"bahamas", "abaco", "acklins", "andros", "berry islands", "bimini", "cat island", "crooked island", "eleuthera", "exuma", "grand bahama", "inagua", "long island", "mayaguana", "new providence", "nassau", "ragged island", "rum cay", "san salvador"},
```
Thank you @ashkulz  for the tip.

I have made the change to the pull request.

Happy coding!  👋

ashkulz commented:
Thanks for the contribution, @steveWDamesJr! I've merged it in [my fork](https://github.com/ashkulz/committers.top) (_see the commit referenced above_) and the [changes are live](https://committers.top/the_bahamas) -- see ashkulz/committers.top@b1d07dfe444468d57ffde75812be5504158d8b20.

steveWDamesJr commented:
> Thanks for the contribution, @steveWDamesJr! I've merged it in [my fork](https://github.com/ashkulz/committers.top) (_see the commit referenced above_) and the [changes are live](https://committers.top/the_bahamas) -- see [ashkulz/committers.top@b1d07df](https://github.com/ashkulz/committers.top/commit/b1d07dfe444468d57ffde75812be5504158d8b20).

Thank you very much @ashkulz. Your efforts in continuing this great work by @lauripiispanen is much appreciated. Thanks also for your commitment to ensuring that 'The Bahamas' now has representation in your fork!

Happy Coding 👋

steveWDamesJr committed:
Create more accurate representation of Bahamas Query

It is realized that many Bahamas Island names are also names of states or countries around the world. To avoid skewed results, I have shortened the QueryPreset to the more unique and inclusive term of 'Bahamas'

steveWDamesJr commented on presets.go:
```diff
@@ -384,7 +384,7 @@ var PRESETS = map[string]QueryPreset{
 	},
 	"the bahamas": QueryPreset{
-		include: []string{"Bahamas", "Abaco", "Acklins", "Andros", "Berry+Islands", "Bimini", "Cat+Island", "Crooked+Island", "Eleuthera", "Exuma", "Grand+Bahama", "Inagua", "Long+Island", "Mayaguana", "New+Providence", "Nassau", "Ragged+Island", "Rum+Cay", "San+Salvador"},
+		include: []string{"Bahamas"},
```
## Hello 👋 

After viewing the results of the live app, it is determined that the results are not an accurate depiction of users from 'The Bahamas' as users from ie. 'Long Island, NY' (which has the same name as Long Island Bahamas) are being grouped together in 'The Bahamas' Rankings. 

This is true of many other Bahamas Island names that share the same name as other states or countries around the world. 

To avoid skewed results, I have shortened the QueryPreset to the more unique and inclusive term of 'Bahamas'. 

Please update this change in your live sites @ashkulz  and @lauripiispanen.

Many thanks 👏

ashkulz replied:
Done, @steveWDamesJr.

steveWDamesJr replied:
Hello @ashkulz 👋 ,

Thank you for your quick action in resolving this issue.

Best,

Steve
